docs/content/en/docs/design/api.md

```diff
@@ -123,15 +123,15 @@ Using `curl` and `HTTP_RPC_PORT=50052`, an example output of a `skaffold dev` ex
 ```
 {{% /tab %}}
 {{% tab "gRPC API" %}}
-To get events from the `gRPC` server, first create [`gRPC` client]({{< relref "/docs/design/api#creating-a-grpc-client" >}})
+To get events from the `gRPC` server, first create [`gRPC` client]({{< relref "#creating-a-grpc-client" >}})
 
 ```golang
 func main() {
   ctx, ctxCancel := context.WithCancel(context.Background())
   defer ctxCancel()
   // `client` is the gRPC client with connection to localhost:50051.
   // See code above to create it
-  logStream, err := client.Events(ctx)
+  logStream, err := client.EventLog(ctx, &empty.Empty{})
```

Comment:
didn't we want to keep this as `client.Events`?

Reply:
yes we should, good catch

Reply:
```suggestion
  logStream, err := client.Events(ctx, &empty.Empty{})
```